### PR TITLE
Feature/sie 17 create grid layout

### PR DESCRIPTION
**Pull request by denglish7**
Created grid layout

### CODE REVIEW

File: src/stories/layouts/Grid/Grid.stories.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ */
+export default {
+    title: 'Example/Grid',
+    layout: Grid,
+};
+
+const Template = (args) => (
+    <Grid numColumns={args.numColumns}>
+        <div className="grid__item">
+            <h1>Test</h1>
```
Comment (alecherryy) [inline src/stories/layouts/Grid/Grid.stories.js:17]: I would updated the content of the sample grid item to be an `h3` and include a paragraph with sample content, like so:
```
<div className="grid__item">
    <h3>Test</h3>
    <p>Lorem ipsum dolor sit amet, aliquip pertinax intellegebat vel ad. 
    Quo diceret comprehensam id, mazim labore instructior et mea.</p>
</div>
```

File: src/stories/layouts/Grid/styles.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+.grid--four {
+    @extend %shared;
+    grid-template-columns: repeat(4, 1fr);
+}
+
+.grid__item {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
```
Comment (alecherryy) [inline src/stories/layouts/Grid/styles.scss:52]: Remove this. We don't want to make assumptions on the display of the grid items.

File: src/stories/layouts/Grid/Grid.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ * @return {object} (
+ *   <Grid numColumns={numColumns}>
+ *      {children}
+ *    </Grid>
+ * )
+ */
+
+export const Grid = ({ children, numColumns }) => {
+    return (
+        <div className={['grid', `${numColumns}`].join(' ').trim()}>
```
Comment (alecherryy) [inline src/stories/layouts/Grid/Grid.js:20]: I like the idea of using the number of columns; I would just update this to `grid--${numColumns}-col` and change the `numColumns` param to a number, which would return the following modifier class: `grid--3-col`.  That way we don't need to spell out numbers which is more likely to lead to typos and then bugs. Example of update:
```
const modifierClass = numColumns ? `grid--${numColumns}-col` : '';
    return (
        <div className={['grid', modifierClass].join(' ').trim()}>
            {children}
        </div>
    );
```

File: src/stories/layouts/Grid/styles.scss [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    @include bp-desktop {
+        grid-template-columns: repeat(4, 1fr);
+        grid-column-gap: $spacing-3;
+    }
+}
+
+.grid--one {
+    @extend %shared;
```
Comment (alecherryy) [inline src/stories/layouts/Grid/styles.scss:29]: You don't need this. Those styles are already present in the `.grid` class.

**Review by alecherryy — CHANGES_REQUESTED** [review]
@denglish7 I left a couple of small suggestions/changes; however, I noticed your grid is not responsive. You might want to double check that!

**Comment by denglish7** [comment]
> @denglish7 I left a couple of small suggestions/changes; however, I noticed your grid is not responsive. You might want to double check that!

**Comment by denglish7** [comment]
> > @denglish7 I left a couple of small suggestions/changes; however, I noticed your grid is not responsive. You might want to double check that!

Thank you! I have made the requested changes and the grid is now responsive with 1-4 columns

**Review by alecherryy — APPROVED** [review]

**Pull request merged**